### PR TITLE
Core doc

### PR DESCRIPTION
**Pull request by psykokwak4**
Migrated current YDK core documentation to rst syntax

### CODE REVIEW

File: ydkgen/printer/doc/doc_printer.py [hunk 1]
```diff
@@ -27,8 +27,9 @@
 
 
 class DocPrinter(object):
-    def __init__(self, ctx):
+    def __init__(self, ctx, language='py'):
```
Comment (ghost) [inline ydkgen/printer/doc/doc_printer.py]: Suggest not providing default value of 'py' for language. Generally, having arguments with default values is not a clean code programming practice and it increases chances of bugs. We may need to clean up the ydkgen/sdk code wherever we have default value aguments

File: ydkgen/printer/meta_data_util.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ def format_range_string(ranges):
     return range_string
 
 
-def get_meta_info_data(prop, property_type, type_stmt, identity_subclasses=None):
+def get_meta_info_data(prop, property_type, type_stmt, identity_subclasses=None, language='py'):
```
Comment (ghost) [inline ydkgen/printer/meta_data_util.py]: > language='py'
> I think language argument should not have a default argument here

File: ydkgen/printer/python/python_bindings_printer.py [hunk 3]
```diff
@@ -219,11 +219,11 @@ def emit_importests(ctx, packages):
 
 
 def emit_module_documentation(ctx, named_element, identity_subclasses):
-    DocPrinter(ctx).print_module_documentation(named_element, identity_subclasses)
+    DocPrinter(ctx, language='py').print_module_documentation(named_element, identity_subclasses)
```
Comment (ghost) [inline ydkgen/printer/python/python_bindings_printer.py]: See above comment about default value of 'py'

Comment (psykokwak4) [reply ydkgen/printer/python/python_bindings_printer.py]: Thanks, will update in next commit.

**Pull request merged**